src/event-notion/NotionCalEvent.ts

```diff
@@ -118,12 +118,18 @@ const filterOrgsResponse = (response: HostFormResponse): StudentOrg[] => {
  * @returns The Interval in which a NotionCalEvent takes place.
  */
 const getEventInterval = (response: HostFormResponse): Interval => {
-  return Interval.fromDateTimes(
+  let interval = Interval.fromDateTimes(
     // Convert from the provided host form responses to the DateTime objects.
     // The format string below SEEMS to work for most events, but I MAY be wrong.
     DateTime.fromFormat(`${response['Preferred date']} ${response['Preferred start time']}`, 'M/d/yyyy h:mm:ss a'),
     DateTime.fromFormat(`${response['Preferred date']} ${response['Preferred end time']}`, 'M/d/yyyy h:mm:ss a'),
   );
+
+  if (interval.toDuration().minutes >= 0) {
+    return interval;
+  }
+
+  throw new RangeError;
```

Comment:
Does RangeError make sense here? I wonder if TypeError makes more sense since this is an invalid interval.

I'm personally okay with keeping RangeError, but we should include a comment explaining why it's a range error.